docs/src/image_loading.md

```diff
@@ -0,0 +1,143 @@
+# Image Loading
+
+## Overview
+
+The Image Loading module is a component of the MCU Runtime SDK designed for managing SOC images. This module provides APIs for:
+
+- Loading SOC images to target components. The SOC images could come from a [Flash Storage](./flash_layout.md) or from another platform (e.g. BMC).
+- Retrieving the SOC Image Metadata (as defined in the [SOC Manifest](https://github.com/chipsalliance/caliptra-sw/blob/main-2.x/auth-manifest/README.md) documentation)
+- Verifying and authenticating the SOC Images.
+
+The diagram below provides an **example** of how the Caliptra subsystem, integrated with custom SOC elements (highlighted in green), facilitates the loading of SOC images to vendor components.
+
+Custom SOC elements:
+
+* **External Flash** : A flash storage containing SOC manifest and the SOC images.
+* **BMC** : An external Board Mangement Controller that provides a service to source the SOC images and manifest.
```

Comment:
I suggest not putting `BMC` under `Custom SOC elements` session.  BMC just interacts with Caliptra subsystem of SoC in streaming boot process, but not part of SOC.

Reply:
That is true, it's not part of the SOC. I'll remove it.